rust/tests/canonicity/chain_discovery.rs

```diff
@@ -26,7 +26,7 @@ async fn gaps() -> anyhow::Result<()> {
 
     // only length 2 is known to be canonical
     assert_eq!(block_parser.total_num_blocks, 26);
-    assert_eq!(block_parser.num_deep_canonical_blocks, 1);
+    assert_eq!(block_parser.num_deep_canonical_blocks, 11);
```

Comment:
These were hand-calculated, from the block directory used in this test. I will need to double check with @Isaac-DeFrain to make sure we both agree about the correctness